foundation-config/src/main/java/io/servicecomb/config/archaius/sources/YAMLConfigurationSource.java

```diff
@@ -16,154 +16,12 @@
 
 package io.servicecomb.config.archaius.sources;
 
-import java.io.File;
-import java.io.InputStream;
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import io.servicecomb.config.ConfigMapping;
-import io.servicecomb.config.YAMLUtil;
-import com.netflix.config.PollResult;
-import com.netflix.config.PolledConfigurationSource;
-
-public class YAMLConfigurationSource implements PolledConfigurationSource {
-
-    private URL[] configUrls = new URL[0];
-
-    private static final String CONFIG_URL = "cse.configurationSource.additionalUrls";
-
-    /**
-     * Default configuration file name to be used by default constructor. This file should
-     * be on the classpath. The file name can be overridden by the value of system property
-     * <code>configurationSource.defaultFileName</code>
-     */
-    private static final String DEFAULT_CONFIG_FILE_NAME = "microservice.yaml";
-
-    private static final String DEFAULT_CONFIG_FILE_FROM_CLASSPATH =
-        System.getProperty("cse.configurationSource.defaultFileName") == null ? DEFAULT_CONFIG_FILE_NAME
-                : System.getProperty("cse.configurationSource.defaultFileName");
-
-    private static final Logger LOGGER = LoggerFactory.getLogger(YAMLConfigurationSource.class);
-
-    private static URL[] createUrls(String... urlStrings) {
-        if (urlStrings == null || urlStrings.length == 0) {
-            throw new IllegalArgumentException("urlStrings is null or empty");
-        }
-        URL[] urls = new URL[urlStrings.length];
-
-        try {
-            for (int i = 0; i < urlStrings.length; i++) {
-                urls[i] = new URL(urlStrings[i]);
-            }
-        } catch (Throwable e) {
-            throw new RuntimeException(e);
-        }
-        return urls;
-    }
-
+// only for unittest
+public class YAMLConfigurationSource extends MicroserviceConfigurationSource {
```

Comment:
io.servicecomb.foundation.ssl.SSLOptionTest
used this class too.
i don't want to copy this to another project.